Solution/source/Menu/Routine.cpp

```diff
@@ -2795,7 +2795,7 @@ void Menu::loops()
 			_JumpAroundMode_::Tick();
 	}
 
-	if (GET_GAME_TIMER() >= delayedTimer)
+	if (GET_GAME_TIMER())
```

Comment:
The removal of the delayedTimer check here is a no-no from me.
You can move your code out of the scope of the check and you don't need to check the game timer.

Reply:
Aight, you affraid of losing some frames... get better PC :p
Ill resolve it in the upcomming days.

Also already added a working rope gun.. ill add that once im satisfies with it  but it working. None glitching..

Been away for 5 years but it still itches...  how have you been mafins..

btw im KiLLerBoy_001 from the old VforVillains..
And darkhorse back in  PS3 days